templates/articles/edit.html

```diff
@@ -51,7 +51,7 @@
   <div class=textarea>
     <label for=body>Body</label>
     <textarea id=body name=body required>{{ article.body }}</textarea>
-    <pre><b>**bold**</b> <i>_italic_</i> #H1 ##H2 ###H3 link: [link text](https://www.google.com) ![alt text](https://github.com/adam-p/markdown-here/raw/master/src/common/images/icon48.png "Logo Title")</pre>
+    <pre><b>**bold**</b> <i>_italic_</i> #H1 ##H2 ###H3 link: [link text](https://www.google.com)<br>Image: ![alt text](https://www/path/to/img.jpg "Logo Title")</pre>
```

Comment:
This code is repeated multiple times. It might be time for a commont layout / a include / a macro

Reply:
That's what I thought.. Afterwards ;-) Ok I imported it now.